.gitattributes

```diff
@@ -5,6 +5,7 @@
 /docs               export-ignore
 /tests              export-ignore
 /lib/Imagine/Test   export-ignore
+/resources/Adobe/Profile Information.pdf export-ignore
```

Comment:
@romainneutron I had also to use `[[:space:]]` instead of the space in the file name